blocks/blockstore/write_cache.go

```diff
@@ -22,6 +22,7 @@ type writecache struct {
 }
 
 func (w *writecache) DeleteBlock(k key.Key) error {
+	defer log.EventBegin(context.TODO(), "Removed block ", &k).Done()
```

Comment:
remove trailing spaces from the string contant, and change the event label to something like `writecache.BlockRemoved`